plugin.cmake

```diff
@@ -14,6 +14,11 @@
 #  limitations under the License.
 ###############################################################################
 
+# support the "if(TEST ...)" command
+if(POLICY CMP0064)
```

Comment:
All right. Would it be reasonable to make the error more transparent in the event that someone uses an old version of CMake and disables the tests, or are we not too concerned about that?